### PR TITLE
Vet Clinic DB: Add Column, Queries and Update Data Table

### PR DESCRIPTION
**Pull request by MoyasiGinko**
### In this milestone, I:
- Created a new branch `query-update-table` from the `development` branch
- Added a column species of type string to the animal's table. 
Screenshot 2023-07-21 121354

- Added: Inside a transaction update the animal's table by setting the species column to unspecified.
Screenshot 2023-07-21 121533

- Added: Inside a transaction update the animals table by setting the species column to Digimon for all animals that have a name ending in mon.
Screenshot 2023-07-21 121701

- Added: Inside a transaction delete all records in the animal's table, then roll back the transaction.
Screenshot 2023-07-21 121730

- Added: Inside a transaction delete all records in the animals table, then commit the transaction.
Screenshot 2023-07-21 121846

- Added: Inside a transaction delete all records in the animals table, then commit the transaction.
image

Thank you, Happy coding 😎

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -10,4 +10,46 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
...
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+SELECT * FROM animals;
+
+
+/*Inside a transaction delete all records in the animals table, then commit the transaction.*/
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT neutered, COUNT(*) FROM animals GROUP BY neutered;
```
Comment (mikethreels) [inline queries.sql]: - [x] in this query you are returning how many animals are neutered vs. how many are not, but the requirements are looking for the check who has more escape attempts.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Moyasi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- good job writing all the queries.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (MoyasiGinko) [reply queries.sql]: Please review, leave a prompt if it is correct?
`SELECT name, neutered, escape_attempts FROM animals WHERE escape_attempts = (SELECT MAX(escape_attempts) FROM animals);`
image

**Review by sumairq — APPROVED** [review]
### Hi @MoyasiGinko , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️good job writing all the queries.
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**